docs/content/latest/explore/ysql-language-features/partitions.md

```diff
@@ -21,25 +21,23 @@ Partitioning is another term for physically dividing large tables in YugabyteDB
 
 Since partitioned tables do not appear nor act differently from the original table, applications accessing the database are not always aware of the fact that partitioning has taken place.
 
-It is recommended to use partitioned tables with a query planner that can decide which tables to access and which ones to ignore based on the table content. 
-
 YSQL supports the following types of partitioning:
 
 - Range partitioning, when a table is partitioned into ranges defined by one or more key columns. In this case, the ranges of values assigned to partitions do not overlap.
 - List partitioning, when a table is partitioned via listing key values to appear in each partition.
 
 ## Declarative Table Partitioning
 
-YSQL allows you to specify how exactly to divide a table. You provide a partitioning method and partition key consiting of a list of columns or expressions. The devided table is called a partitioned table, and the resulting tables are called partitions. When you insert rows into a partitioned table, they are redirected to a partition depending on the value of the partition key.
+YSQL allows you to specify how exactly to divide a table. You provide a partitioning method and partition key consiting of a list of columns or expressions. The divided table is called a partitioned table, and the resulting tables are called partitions. When you insert rows into a partitioned table, they are redirected to a partition depending on the value of the partition key.
 
-You can partition partitions, in which case they woud have their own distinct indexes, constraints, and default values.
+You can nest partitions, in which case they woud have their own distinct indexes, constraints, and default values.
 
-A regular table cannot become a partitioned table, just as a partitioned table cannot become a regular table. That said, you can add a regular or partitioned table that has data as a partition of a partitioned table, and you can remove a partition from a partitioned table making it a standalone table.
+A regular table cannot become a partitioned table, just as a partitioned table cannot become a regular table. That said, YSQL supports adding a regular table that has the same schema as that of the partitioned table to be attached as one of the partitions of the partitioned table. Conversely, a partition can be detached from the partitioned table, in which case it can behave as a regular table that is not part of the partitioning hierarchy.
```

Comment:
My bad, apologies. Can you change the sentence "YSQL supports adding a regular table" into "YSQL allows attaching a regular table (provided it has the same schema as that of the partitioned table) as a partition to the partitioned table.

Reply:
Changed